xla/tests/BUILD

```diff
@@ -2352,10 +2362,12 @@ xla_cc_test(
         "//xla/service:platform_util",
         "//xla/service/cpu:cpu_compiler",
         "//xla/service/gpu:gpu_compiler",
-        "//xla/stream_executor",
         "//xla/stream_executor/cuda:cuda_platform_id",
+        "//xla/stream_executor/rocm:rocm_platform_id",
```

Comment:
I think this should be `if_rocm_is_configured`, (and same for cuda_platform_id, shoul dbe `if_cuda_is_configured`). Feel free to make or not make the cuda_platform_id change as that's immaterial to this PR

Reply:
Agree, please change it back to put these dependencies behind if_rocm_is_configured/if_cuda_is_configured guards, respectively.

Reply:
Thanks! Done